diff --git a/src/parser/gjs-gts-parser.js b/src/parser/gjs-gts-parser.js
index ce9702f50f219d8933995b7fe236245734b050fb..ddace9aba37693258f01eefa828b120cf1513e88 100644
--- a/src/parser/gjs-gts-parser.js
+++ b/src/parser/gjs-gts-parser.js
@@ -1,5 +1,5 @@
 const tsconfigUtils = require('@typescript-eslint/tsconfig-utils');
-const babelParser = require('@babel/eslint-parser');
+const babelParser = require('@babel/eslint-parser/experimental-worker');
 const { registerParsedFile } = require('../preprocessor/noop');
 const {
   patchTs,
